markdown/projects/mr-robot/major-code/arduino.rst

```diff
@@ -0,0 +1,113 @@
+Arduino
+=======
+
+.. code:: C
+
+    void LpwmCb(const std_msgs::Int32& pwm){
+
+        if(pwm.data == 0) { ledcWrite(ledChannel1, 0); ledcWrite(ledChannel2, 0); }
+        if(pwm.data > 0)  { ledcWrite(ledChannel1, abs(pwm.data)); ledcWrite(ledChannel2, 0); }
+        if(pwm.data < 0)  { ledcWrite(ledChannel1, 0); ledcWrite(ledChannel2, abs(pwm.data));}
+    }
+
+This code defines two callback functions, LpwmCb and RpwmCb, which take a std_msgs::Int32 object as input. The functions control the speed of four motors by writing PWM signals using the ledcWrite function.
```

Comment:
Four motors were not used. Write two instead.

Reply:
Add Inline code for highlighting the different code components: LpwmCb, std_msgs